_includes/head.html

```diff
@@ -1,7 +1,11 @@
 <head>
   <meta charset='utf-8' />
   <meta http-equiv="X-UA-Compatible" content="chrome=1" />
-  <meta name="description" content="Up For Grabs : Jump in!" />
+  <meta property="og:url"         content="http://up-for-grabs.net" />
+  <meta property="og:type"        content="product" />
```

Comment:
I don't think we're a `product`. I'd stick with `website`, or drop this altogether.

> If you don't specify a type,the default is `website`.